docs/main-concepts/how-to-define-a-fetch.md

```diff
@@ -0,0 +1,352 @@
+---
+id: how-to-define-a-fetch
+title: How to define a fetch
+sidebar_label: How to define a fetch
+---
+
+The `defineFetch` API has the following shape:
+
+`makePersonFetch.js`
+
+```js
+import { defineFetch } from 'resift';
+
+const makePersonFetch = defineFetch({
+  displayName: /* {display name} */,
+  make: (/* {make args} */) => ({
+    key: /* {key} */,
+    request: (/* {request args} */) => ({ http }) => {
+      /* {request body} */
+    },
+  }),
+});
+
+export default makePersonFetch;
+```
+
+There is:
+
+- the `displayName`,
```

Comment:
I would just remove this entire section, you're already describing the args below